catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -173,6 +180,12 @@ public void setHiddenAttributes(List<String> hiddenAttributes) {
         this.hiddenAttributes = hiddenAttributes;
     }
 
+    public void setAttributeDescriptions(List<String> attributeDescriptions) {
+        this.attributeDescriptions = attributeDescriptions.stream()
+                .map(str -> str.split("="))
+                .collect(Collectors.toMap(list -> list[0].trim(), list -> list[1].trim()));
```

Comment:
What happens if there is no `=` in the attributeDescription?  Might be worth a check to avoid the index out of bounds exception.